app/model.js

```diff
@@ -71,4 +71,127 @@ const fetchArticleById = (params) =>
 }
 
 
-module.exports = {fetchTopics, fetchArticles, fetchArticleById}
\ No newline at end of file
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+const fetchUserComment = (params, requestBody) =>
+{
+    if(!/[0-9]+/g.test(params))
```

Comment:
regex isn't needed here PSQL will throw an error if the Article ID is not a number